File: Banco de dados nao relacional MongoDB/R/conexao.R
install.packages("mongolite")
library(mongolite)

conexao = mongo(collection = "posts", db = "dbmidias", url = "mongodb://localhost")

dados = conexao$find()
class(dados)
dados

#busca
dados = conexao$find('{"nome":"José"}')
dados

#fecha conexão
rm(conexao)
gc()

File: Banco de dados nao relacional MongoDB/Python/conexao.py
from pymongo import MongoClient

#conexão local padrão
cliente = MongoClient('mongodb://localhost:27017/')
#banco dbmidias
db = cliente.dbmidias
#coleção posts
conexao = db.posts

#um documento
print(conexao.find_one())

print(conexao.find_one({"nome":"Maria Silva"}))

#todos
for conexao in conexao.find():
    print(conexao)

cliente.close()